editoast/src/models/errors/mod.rs

```diff
@@ -69,34 +71,43 @@ enum InfraErrorType {
 }
 
 impl InfraError {
-    fn new_invalid_reference<T: AsRef<str>>(field: T, reference: ObjectRef) -> Self {
+    fn new_invalid_reference<T: AsRef<str>>(id: String, field: T, reference: ObjectRef) -> Self {
```

Comment:
```suggestion
    fn new_invalid_reference<T: AsRef<str>>(id: T, field: T, reference: ObjectRef) -> Self {
```

Reply:
I thought about this, but this change force me to have the same type for the id and the field, in editoast, the ID are Strings and field are STR, so if I put T here I have to convert id into str, or field into string, not sure is very useful ? maybe create a second type U: AsRef<str>> for the id ?

Reply:
Indeed you have to create a second generic type.